### ISSUE TITLE
A question about curriculum learning

### ISSUE THREAD

**Issue by gyccccccccc** [Derekabc/MARL_CAVs#35]
<img width="396" alt="屏幕截图 2024-02-22 153042" src="https://github.com/DongChen06/MARL_CAVs/assets/142129312/2aec266f-23e1-4bda-8733-4d287ea24fb3">
Hi!
Sorry to bother you again, I want to know whether curriculum learning in the paper means to train the model three times, in 3 different traffic modes?
If i'm right,  Could you please tell me how can I continue training the pre-trained model in another modes(eg. Medium Mode or Hard)? how can I switch it to Medium mode?
I've trained the model in the easy one, but I don't know how to do next.

**Comment by gyccccccccc** [comment]
thanks， i’ve figure it out！

**Comment by xbv001** [comment]
Hi! May I know your code runtime environment? I ran the code on the Windows system, but there were some issues. Excuse me, are you also running code under Python 3.6?

**Comment by gyccccccccc** [comment]
> Hi! May I know your code runtime environment? I ran the code on the Windows system, but there were some issues. Excuse me, are you also running code under Python 3.6?

Hi，I also ran the code on Windows, but I modified part of the code
eg
"
one_hot = np.zeros((len(index), dim))
one_hot[np.arange(len(index)), index] = 1.
"
to
"
ss = str(index)
one_hot = np.zeros((len(ss), dim))
one_hot[np.arange(len(ss)), index] = 1.
"

**Comment by gyccccccccc** [comment]
and I ran the code under Python 3.6
